
public class ExceptionDemo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			int i = 9 / 7;
		}
		catch(Exception e){
			System.out.println("e");
		}
		
		System.out.println("Bye");
	}

}
